app/views/hostgroups/_form.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= javascript 'host_edit', 'host_edit_interfaces' %>
 <%= form_for @hostgroup, :html => {:class => 'hostresource-form form-horizontal well', :data => {:id => @hostgroup.try(:id)}} do |f| %>
   <%= base_errors_for @hostgroup %>
+  <%= react_component('ResourceErrors', host_errors(@hostgroup).to_json) %>
```

Comment:
We will most likely use even the values of the lookups, right?
Should we then just pass the lookup values to the slot? as most likely all of the Fills would somehow manipulate with the values right?

Reply:
The only responsibility of this component is to load errors into store, the component adds nothing to the page and there are no Slots or Fills added.

If other components need any values, passing the them through slot is a possibility but no such component is in this PR.